### PR TITLE
Doc strings within 80 char limit for hbut.el

### PR DESCRIPTION
**Pull request by matsl**
## What

Doc strings within 80 char limit for hbut.el

## Why

Extracted from longer PR for easier review. It is still quite long. Let me know if I should split it further.

### CODE REVIEW

File: hbut.el [hunk 1]
```diff
@@ -1330,7 +1341,7 @@ calls `hbut:modify-syntax'.")
   nil)
 
 (defun    hbut:outside-comment-p ()
-  "Return t if within a programming language buffer and prior regexp match is outside a comment, else nil."
+  "True within a programming language buffer and prior match is outside a comment."
```
Comment (rswgnu) [inline hbut.el:1344]: "True if in a programming mode and regexp match is outside a comment, else nil."

File: hbut.el [hunk 2]
```diff
@@ -1654,7 +1671,7 @@ expression which matches an entire button string."
   (hbut:map but-func ibut:label-start ibut:label-end))
 
 (defun    ibut:rename (old-lbl new-lbl)
-  "Rename a label preceding the text of a Hyperbole implicit button in the current buffer from OLD-LBL to NEW-LBL.
+  "Rename an implicit button label in the current buffer from OLD-LBL to NEW-LBL.
```
Comment (rswgnu) [inline hbut.el:1674]: "Change an implicit button name in the current buffer from OLD-LBL to NEW-LBL.

File: hbut.el [hunk 3]
```diff
@@ -1752,18 +1771,19 @@ positions at which the button label delimiter begins and ends."
 
 (defun    ibut:map (but-func &optional _start-delim _end-delim
 			     regexp-match include-delims)
-  "Apply BUT-FUNC to the labeled implicit buttons in the visible part of the current buffer.
-If REGEXP-MATCH is non-nil, only buttons which match this argument are
-considered.
+  "Apply BUT-FUNC to the labeled implicit buttons.
```
Comment (rswgnu) [inline hbut.el:1774]: "Apply BUT-FUNC to the visible, named implicit buttons.
If REGEXP-MATCH is non-nil, only buttons which match this argument
are considered.

File: hbut.el [hunk 4]
```diff
@@ -1840,9 +1861,10 @@ Return the symbol for the button, else nil."
 		lbl-key))
 
 (defun    ibut:at-to-name-p (&optional ibut)
-  "If point is on an implicit button, optional IBUT, move to the start of its name, if any (past opening delimiter).
-When found, set the name and lbl-key properties of IBUT.
-Return t if name is found, else nil."
+  "Move to start of an implicit buttons name, if any (past opening delimiter).
+Point is on an implicit button or optional IBUT.  When found, set
+the name and lbl-key properties of IBUT.  Return t if name is
+found, else nil."
```
Comment (rswgnu) [inline hbut.el:1867]: "If point is on an implicit button, optional IBUT, move to the start of its name.
If name is found, leave point after its opening delimiter and set the name
and lbl-key properties of IBUT.  Return t if name is found, else nil."

File: hbut.el [hunk 5]
```diff
@@ -1864,10 +1886,14 @@ Return t if name is found, else nil."
     move-flag))
 
 (defun    ibut:to-name (lbl-key)
-  "Find the nearest implicit button with LBL-KEY (a label or label key) within the visible portion of the current buffer and move to the start of its delimited button name (after opening delimiter).
-This will find an implicit button if point is within its name or text
-or if LBL-KEY is a name/name-key of an existing implicit button.  It
-will not find other unnamed implicit buttons.
+  "Move to the start of the nearest implicit buttons delimited button name.
```
Comment (rswgnu) [inline hbut.el:1889]: "Move to the name of the nearest named implicit button matching LBL-KEY.

File: hbut.el [hunk 6]
```diff
@@ -1893,10 +1919,13 @@ Return the symbol for the button if found, else nil."
    lbl-key))
 
 (defun    ibut:to-text (lbl-key)
-  "Find the nearest implicit button with LBL-KEY (a label or label key) within the visible portion of the current buffer and move to within its button text.
-This will find an implicit button if point is within its name or text
-or if LBL-KEY is a name/name-key of an existing implicit button.  It
-will not find other unnamed implicit buttons.
+  "Move to within nearest implicit button text.
```
Comment (rswgnu) [inline hbut.el:1922]: "Move to the text of the nearest implicit button matching LBL-KEY.

File: hbut.el [hunk 7]
```diff
@@ -1944,11 +1973,14 @@ Return the symbol for the button if found, else nil."
   "String matching the end of a Hyperbole implicit button label.")
 
 (defvar   ibut:label-separator " "
-  "String inserted immediately after a newly created implicit button label to separate it from the implicit button text.
-See also `ibut:label-separator-regexp' for all valid characters that may manually inserted to separate an implicit button label from its text.")
+  "String inserted immediately after a newly created implicit button label.
```
Comment (rswgnu) [inline hbut.el:1976]: "String inserted immediately after a newly created implicit button name.

File: hbut.el [hunk 8]
```diff
@@ -1944,11 +1973,14 @@ Return the symbol for the button if found, else nil."
...
+This separates it from the implicit button text.  See also
+`ibut:label-separator-regexp' for all valid characters that may
+manually inserted to separate an implicit button label from its
+text.")
 
 (defvar   ibut:label-separator-regexp "\\s-*[-:=]*\\s-+"
-  "Regular expression that separates an implicit button label from its implicit button text.")
+  "Regular expression that separates an implicit button label from its button text.")
```
Comment (rswgnu) [inline hbut.el:1983]: "Regular expression that separates an implicit button name from its button text.")

File: hbut.el [hunk 9]
```diff
@@ -2004,7 +2036,7 @@ type for ibtype is presently undefined."
 (defalias 'ibtype:create #'defib)
 
 (defun ibtype:activate-link (referent)
-  "Activate a Hyperbole implicit link `referent', either a key series, a URL or a path."
+  "Activate an implicit link REFERENT, either a key series, a URL or a path."
```
Comment (rswgnu) [inline hbut.el:2039]: "Activate an implicit link REFERENT, either a key series, a url or a path."

File: hbut.el [hunk 10]
```diff
@@ -2042,8 +2077,8 @@ the whole button text (\\\\&) or any numbered grouping from
 TEXT-REGEXP, e.g. \\\\1, may be referenced in the LINK-EXPR to
 form the link referent.
 
-Here is a sample use case.  Let's create a button type whose
-buttons perform a grep-like function over a current repo's git
+Here is a sample use case.  Let us create a button type whose
+buttons perform a grep-like function over a current repositorys git
```
Comment (rswgnu) [inline hbut.el:2081]: Here is a sample use case.  Create a button type whose buttons
perform a grep-like function over a current repository's git

**Review by rswgnu — CHANGES_REQUESTED** [review]

File: hbut.el (hunk 9 above)
Comment (matsl) [reply hbut.el:2039]: This is the same text as given. Did you forget to do the change you wanted?

**Review by rswgnu — APPROVED** [review]

**Pull request merged**